### DISCUSSION TITLE
Orphaned edge

### DISCUSSION THREAD

**Discussion by Copca92** [bcakmakoglu/vue-flow#696] [Q&A]
When I delete some line or add a node I lose some lines and it gives me a warning of Orphaned edge

**Comment by bcakmakoglu** [comment]
Can you share an example of your code? Or maybe even a sandbox? 😄 

The warning is shown (only in development, in production builds this warning will not appear anymore), when an edge is forcefully removed from the state because either their source node or target node does not exist anymore and we can't have edges floating in "mid-air" not being connected to a handle.

If that is intentional, you can basically ignore the warning and just use it as a hint that an edge was cleaned up due to missing source/target.
